src/Utilities/overviewRows.js

```diff
@@ -18,7 +18,7 @@ import JSZip from 'jszip';
 import flatten from 'lodash/flatten';
 import { treeTable } from '@redhat-cloud-services/frontend-components/components/TreeTable';
 
-const indexToKey = [ 'title', 'appName', 'version' ];
+const indexToKey = [ 'title', 'title', 'appName', 'version' ]; // pf indexes from 1 not 0
```

Comment:
I think you can leave the first empty
```suggestion
const indexToKey = [ '', 'title', 'appName', 'version' ]; // pf indexes from 1 not 0
```